book/embeds.md

```diff
@@ -902,6 +917,12 @@ a "seamless" iframe whose layout coordinates with its parent frame.
 ``` {.python}
 class BlockLayout:
     # ...
+
+    def layout_mode(self):
+        # ...
+        elif self.node.tag in {"input", "img", "iframe"}:
```

Comment:
Should this include "button"?

Reply:
Uhh, I think so, though it should only matter in pretty weird cases since `button` elements typically have children.